<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WQ_Analysing_Widget</class>
 <widget class="QWidget" name="WQ_Analysing_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>363</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>306</width>
    <height>363</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>306</width>
    <height>363</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QWidget { background-color: white}

.QScrollArea{ background-color: white; border: 1px solid gray}

.QGroupBox{ background-color: white; border: 1px solid gray}</string>
  </property>
  <widget class="QWidget" name="widgetTotalNew" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>306</width>
     <height>363</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>306</width>
     <height>363</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>306</width>
     <height>363</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QComboBox" name="comboDatos">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>296</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxExpress">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>52</y>
      <width>296</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Express</string>
    </property>
    <widget class="QPushButton" name="botonTiempo">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Imagenes/Time2.png</normaloff>Imagenes/Time2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="botonCompleto">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Imagenes/Complete2.png</normaloff>Imagenes/Complete2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAdvanced">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>127</y>
      <width>296</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Advanced</string>
    </property>
    <widget class="QLabel" name="labelStart">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>70</y>
       <width>60</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinDurationSegundos">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>100</y>
       <width>45</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinDurationMicros">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>100</y>
       <width>55</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelStartMicros">
     <property name="geometry">
      <rect>
       <x>273</x>
       <y>70</y>
       <width>18</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>us</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDurationSegundos">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>100</y>
       <width>10</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>s</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStartMilis">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>70</y>
       <width>18</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinDurationMilis">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>100</y>
       <width>55</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelStartSegundos">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>70</y>
       <width>10</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>s</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDuration">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>100</y>
       <width>60</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDurationMicros">
     <property name="geometry">
      <rect>
       <x>273</x>
       <y>100</y>
       <width>18</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>us</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDurationMilis">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>100</y>
       <width>18</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinStartMilis">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>70</y>
       <width>55</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinStartSegundos">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>70</y>
       <width>45</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinStartMicros">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>70</y>
       <width>55</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboTipoAnalisis">
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Time Series</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Arrival Probability</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Autocorrelation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>H vs m</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBoxYLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>145</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Y axis logarithmic</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxXLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>145</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>X axis logarithmic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botonAddCurve">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>132</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add Curve</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Imagenes/Add1.png</normaloff>Imagenes/Add1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="comboChart">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>160</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>New Chart</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBoxReplace">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>80</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Replace</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
